redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -465,10 +565,12 @@ inline void
     getHwIsolationStatus(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
                          const sdbusplus::message::object_path& resourceObjPath)
 {
-    crow::connections::systemBus->async_method_call(
-        [aResp, resourceObjPath](
-            boost::system::error_code ec,
-            const std::variant<std::vector<std::string>>& vEndpoints) {
+    sdbusplus::asio::getProperty<std::vector<std::string>>(
+        *crow::connections::systemBus, "xyz.openbmc_project.ObjectMapper",
+        resourceObjPath.str + "/event_log", "xyz.openbmc_project.Association",
+        "endpoints",
+        [aResp, resourceObjPath](boost::system::error_code ec,
+                                 const std::vector<std::string>& vEndpoints) {
```

Comment:
Similar to the above case, this can be done with `dbus::utility::getAssociationEndPoints()`